### PR TITLE
feat: Add initial GraphQL configuration

### PR DESCRIPTION
**Pull request by vxcamiloxv**
Add initial GraphQL configuration using [actix](https://actix.rs/) and https://github.com/async-graphql/async-graphql, include linter and format rules

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+path = "src/main.rs"
+
+[dependencies]
+actix-web = "3"
+async-graphql = "2.9.9"
+async-graphql-actix-web = "2.9.9"
+serde = { version = "1.0.126", features = ["derive"] }
+serde_json = "1.0.64"
+dotenv = "0.15.0"
+clap = "2.33.3"
```
Comment (Tremerex) [inline Cargo.toml:19]: extra line!

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# top-most EditorConfig file
+root = true
+
+[*]
+indent_size = 4
```
Comment (Tremerex) [inline .editorconfig:5]: 4???

File: clippy.toml [hunk 3]
```diff
@@ -0,0 +1 @@
+cognitive-complexity-threshold = 25
```
Comment (Tremerex) [inline clippy.toml:1]: extra line!

File: src/defaults.rs [hunk 4]
```diff
@@ -0,0 +1,4 @@
+//! Default values for the CLI and server
+
+pub const PORT: &str = "5050";
```
Comment (Tremerex) [inline src/defaults.rs:3]: Why &str??? I think is better no use references values here!

File: .editorconfig (hunk 2 above)
Comment (vxcamiloxv) [reply .editorconfig:5]: yes, this is like the default style in rust https://github.com/rust-dev-tools/fmt-rfcs/blob/master/guide/guide.md#indentation-and-line-width

File: src/defaults.rs (hunk 4 above)
Comment (vxcamiloxv) [reply src/defaults.rs:3]: `&str` is immutable so it make sense for a constant also it's the preferred way to pass string, as string also we need to do `const PORT: String = "5050".to_string()` and in for example `cli.rs` `POST.as_str()` so it require additional steps https://www.ameyalokare.com/rust/2017/10/12/rust-str-vs-String.html

File: clippy.toml (hunk 3 above)
Comment (vxcamiloxv) [reply clippy.toml]: done https://github.com/123L-it/vit-graphql/pull/1/commits/fd8578359a13dac436987d1f6b18ee9dde39bc72

File: Cargo.toml (hunk 1 above)
Comment (vxcamiloxv) [reply Cargo.toml]: done https://github.com/123L-it/vit-graphql/pull/1/commits/fd8578359a13dac436987d1f6b18ee9dde39bc72

File: .editorconfig (hunk 2 above)
Comment (Tremerex) [reply .editorconfig:5]: ok!

File: src/defaults.rs (hunk 4 above)
Comment (Tremerex) [reply src/defaults.rs:3]: you can use the `String::from` to avoid the `to_string` but what do you think about to use the cast `as_str` or the `unwrap`???

**Review by almahirpm — APPROVED** [review]
Awsome!

**Review by Tremerex — APPROVED** [review]

**Pull request merged**